packages/Webkul/SocialLogin/src/Repositories/CustomerSocialAccountRepository.php

```diff
@@ -71,7 +71,7 @@ public function findOrCreateCustomer($providerUser, $provider)
         if ($account) {
             return $account->customer;
         } else {
-            $customer = $this->customerRepository->findOneByField('email', $providerUser->getEmail());
+            $customer = $providerUser->getEmail()?$this->customerRepository->findOneByField('email', $providerUser->getEmail()):null;
```

Comment:
Hi,

Please add spaces before and after '?' and ':'.

`$providerUser->getEmail() ? $this->customerRepository->findOneByField('email', $providerUser->getEmail()) : null;`